cmd/utils.go

```diff
@@ -1686,13 +1686,54 @@ func KubeDelete(log *LoggingConfig, fileToApply string, namespace string, dryrun
 	return kerr
 }
 
+//KubeGetNodePortURLIBMCloud issues several kubectl commands and prints the concatenated URL
+func KubeGetNodePortURLIBMCloud(log *LoggingConfig, service string, namespace string, dryrun bool) (url string, err error) {
+	kargs := append([]string{"pod"})
```

Comment:
you can remove the append() wrapping `[]string{"pod"}`

Reply:
Sure will do that.

Reply:
Done